client/rest-high-level/build.gradle

```diff
@@ -27,11 +27,24 @@ group = 'org.elasticsearch.client'
 archivesBaseName = 'elasticsearch-rest-high-level-client'
 
 publishing {
-    publications {
```

Comment:
This didn't line up with the rest of the project and was confusing me so I fixed it.